akka-docs-new/src/test/scala/tutorial_3/DeviceGroupSpec.scala

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package tutorial_3
+
+import akka.actor.PoisonPill
+import akka.testkit.{ AkkaSpec, TestProbe }
+
+import scala.concurrent.duration._
+
+class DeviceGroupSpec extends AkkaSpec {
+
+  "DeviceGroup actor" must {
+
+    "be able to register a device actor" in {
+      val probe = TestProbe()
+      val groupActor = system.actorOf(DeviceGroup.props("group"))
+
+      probe.send(groupActor, DeviceManager.RequestTrackDevice("group", "device1"))
```

Comment:
I prefer 

    groupActor.tell(DeviceManager.RequestTrackDevice("group", "device1"), probe.ref)

because it's using ordinary ActorRef message sending instead of introducing the (backward) concept of `probe.send`